src/content/app/genome-browser/components/track-settings-panel/track-config-views/RegularTrackConfig.tsx

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
Is this file used anywhere?

Reply:
Yes, for all the non gene and transcript tracks